### DISCUSSION TITLE
Neural Magic's DeepSparse - Ultralytics YOLOv8 Docs

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#2597] [Docs]
# Neural Magic's DeepSparse - Ultralytics YOLOv8 Docs

Learn how to deploy YOLOv5 with DeepSparse to achieve exceptional CPU performance close to GPUs, using pruning, and quantization.<br>

https://docs.ultralytics.com/yolov5/tutorials/neural_magic_pruning_quantization/

**Comment by scraus** [comment]
It is giving me an error. Please helpp

![image](https://github.com/ultralytics/ultralytics/assets/91582176/e86263dd-2fbc-401a-82c8-85faaf388fcc)

**Reply by aqibriaz95** [reply]
Seems like version incompatibility issue.

**Reply by pderrenger** [reply]
@aqibriaz95 it looks like you're encountering a version incompatibility issue. To resolve this, please ensure that you have the correct version of the required libraries installed. You can do this by following the installation instructions in our documentation, which will guide you through setting up the correct environment for YOLOv8.

If the problem persists after checking the library versions, please provide more details about the error message you're receiving, and I'll be happy to assist you further. 🛠️
